scripts/github-tag

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# exit when any command fails
+set -e
+
+# Find the last tag in develop
+latest=$(git tag  | grep -E '^[0-9]' | sort -V | tail -1)
+
+# Display the latest tag
+echo $latest
+
+# Use the last tag on master and increase minor by 1.
+version=(${latest//./ })
+major=${version[0]}
+minor=${version[1]}
+patch='0'
+TAG=${major}.$((minor+1)).${patch}
+
+# Display the new tag version.
+echo $TAG
+
+# Create post data
+generate_post_data()
+{
+  cat <<EOF
+{
+    "tag_name":"${TAG}",
+    "target_commitish": "master",
+    "name": "${TAG}",
+    "body": "Please paste notes from changelog."
```

Comment:
the copy and past of the release tag body could potentially be automated, maybe we can follow up on this